docs/faq.md

```diff
@@ -6,4 +6,4 @@ Unfortunately, no. Those problem types cannot be used in a `customresponse` prob
 
 - Does the grader work with python 2 or python 3?
```

Comment:
I edited this FAQ, but I don't know if it is still required to have that question.

What do you think @jolyonb?

Reply:
Heh, yeah, can probably remove that now lol!